spec/support/schemas/claims_api/v2/errors/default.json

```diff
@@ -0,0 +1,41 @@
+{
+  "required": ["errors"],
+  "properties": {
+    "errors": {
+      "type": "array",
+      "items": {
+        "additionalProperties": false,
+        "required": ["title", "detail"],
+        "properties": {
+          "title": {
+            "type": "string",
+            "description": "HTTP error title"
+          },
+          "detail": {
+            "type": "string",
+            "description": "HTTP error detail"
+          },
+          "code": {
+            "type": "string",
+            "description": "HTTP error code"
+          },
+          "status": {
+            "type": "string",
+            "description": "HTTP error code"
+          },
```

Comment:
Let's remove these (`code` and `status`) from the schema if we're not planning to use them for now.

Reply:
Should I remove "code" and "status" from all other responses in v2?

Reply:
@jw81 I think leaving them in is fine, for now. Since we're only intercepting the token validation error to start with, other errors should still have those properties, right? So having them here but not in `required` seems correct to me.